doc/commands/search.8.rst

```diff
@@ -0,0 +1,58 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _search_command_ref-label:
+
+###############
+ Search Command
+###############
+
+Synopsis
+========
+
+``dnf5 search [options] <pattern>...``
+
+
+Description
+===========
+
+The ``search`` command in ``DNF5`` is used for searching packages by matching
+given keywords from the user against various metadata. By default the command
+searches all requested keys in `Name` or `Summary` fields from the RPM
+package metadata.
```

Comment:
As opposed to the dnf4 documentation, there is not mentioned case-sensitivity and the fact that "AND operation" is used. The other is shown in the examples and it's not hard to figure it out after one use, but I think it would be still worth mentioning here as well.

Reply:
Sure, this should be there, thanks!